setup.py

```diff
@@ -39,9 +39,9 @@
 version = f'{TI_VERSION_MAJOR}.{TI_VERSION_MINOR}.{TI_VERSION_PATCH}'
 if project_name == 'taichi-nightly':
     from datetime import date
-    today = date.today().strftime('%Y_%m_%d')
+    today = date.today().strftime('%Y%m%d')
     commit_sha = os.getenv('COMMIT_SHA')
-    version += f'.{today}.{commit_sha}'
+    version += f'.dev{today}'
```

Comment:
Do we still want `commit_sha`? ;)

Reply:
sorry just saw the PR summary, does it work if we keep the first 6 digits of commit sha?

Reply:
I believe not, because as PEP440 mentions, the version must be in a meaningful order. Apparently, commit hash does not guarantee this property...

Reply:
And I am already only keeping the first 6 digits XD.